Blinding/src/TrackDigiExtractor_module.cc

```diff
@@ -0,0 +1,122 @@
+// Ed Callaghan
+// Extract digis which participate in a prefit track
+// September 2024
+
+// stl
+#include <string>
+
+// art
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Principal/Event.h"
+
+// cetlib_except
+#include "cetlib_except/exception.h"
+
+// fhiclcpp
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Comment.h"
+#include "fhiclcpp/types/Name.h"
+
+// mu2e
+#include "Offline/RecoDataProducts/inc/KalSeed.hh"
+#include "Offline/RecoDataProducts/inc/StrawHit.hh"
+#include "Offline/RecoDataProducts/inc/StrawDigi.hh"
+#include "Offline/RecoDataProducts/inc/TrkStrawHitSeed.hh"
+
+namespace mu2e{
+  class TrackDigiExtractor: public art::EDProducer{
+    public:
+      struct Config{
+        // track
+        fhicl::Atom<art::InputTag> kalseed_tag{
```

Comment:
Is it conceivable this module will need to coherently process multiple KalSeed collections (ie multiple particle hypotheses)?

Reply:
It is conceivable, but IMO unlikely for the conversion-blinding use case where we really only want electron-like tracks. Maybe we'll determine that selecting another stream is useful for some book-keeping purpose, but I doubt that we'd want to collate that with the electron-likes at this stage.